projects/canopy/src/lib/promo-card/promo-card/promo-card.component.scss

```diff
@@ -7,6 +7,11 @@
 
   @media (min-width: $promo-card-inline-breakpoint) {
     min-height: var(--promo-card-image-dimension);
+    display: flex;
+    align-items: center;
+  }
+  @include lg-breakpoint('lg') {
+    display: unset;
```

Comment:
I think `unset` this might have compatibility issues with IE11, would it work to use `inherit` or just set it back to `block`?

Reply:
Yeah, I updated the code with `display: block`